bwa3_comp_ace/CfgVehicles.hpp

```diff
@@ -33,6 +34,15 @@ class CfgVehicles {
 
     class Tank_F: Tank {};
 
+    class Car_F: Car {
+        class Turrets {
+            class MainTurret: NewTurret {
+                class Turrets {
+                };
```

Comment:
1. This Turrets sub class does not exist and even if it did, the Eagle has no commander turret.
2. This syntax would remove all inherited entries from the class. The correct way to inherit would be to just put a semi-colon without curly brackets.

Delete line 40-41.